mobile/lib/screens/settings/settings_page.dart

```diff
@@ -127,28 +126,6 @@ class _SettingsPageState extends State<SettingsPage>
                           ),
                         ),
                         divider,
-                        Card(
```

Comment:
@katendejericho5 
Also remember to shift the card to the general Profile Page for Guest users.